examples/nlp/nmt_tutorial.py

```diff
@@ -44,7 +45,12 @@
 parser.add_argument("--max_seq_length", default=256, type=int)
 parser.add_argument("--label_smoothing", default=0.1, type=float)
 parser.add_argument("--beam_size", default=4, type=int)
-parser.add_argument("--tokenizer_model", default="bpe8k_yttm.model", type=str)
+# pass a YouTokenToMe model to YouTokenToMeTokenizer for en&de
+parser.add_argument("--src_tokenizer_model",
+                    default="bpe8k_yttm.model", type=str)
+# pass a vocabulary text file to CharTokenizer for zh
+parser.add_argument("--tgt_tokenizer_model",
+                    default="zh_vocab.txt", type=str)
```

Comment:
the default or tgt_tokenizer_model and tgt_lang should be consistent. right now tgt_lang is de but tgt_tokenizer_model is zh

Reply:
@chiphuyen fixed